RELEASE-NOTES.txt

```diff
@@ -1,5 +1,8 @@
 2.11
 -----
+* Added linking between notes [https://github.com/Automattic/simplenote-android/pull/1145]
+* Fixed toggling tab-indented checkboxes [https://github.com/Automattic/simplenote-android/pull/1148]
+* Fixed adding or editing a tag on Android 6 [https://github.com/Automattic/simplenote-android/pull/1157]
```

Comment:
Unfortunately, we haven't put automation on the Fastlane lane that makes the code freeze to add a new release note entry.

To avoid conflicts with the `release/2.11` branch and to keep a "correct" release notes log, I suggest doing this. Really, I should have though of that when I pushed the branch, but I didn't notice it 😞 

```suggestion

2.10.1
-----

* Fixed adding or editing a tag on Android 6 [https://github.com/Automattic/simplenote-android/pull/1157]
```

Reply:
I reverted the commit changing the descriptions in 691c5243 and I moved the tag fix description from 2.11 to 2.10.1 in 4b2fba6a.